app/src/main/java/com/segmentify/segmentifysdk/EventActivity.java

```diff
@@ -1,24 +1,17 @@
 package com.segmentify.segmentifysdk;
 
-import android.support.annotation.NonNull;
-import android.support.v7.app.AppCompatActivity;
 import android.os.Bundle;
-import android.util.Log;
+import android.support.v7.app.AppCompatActivity;
```

Comment:
why did we add this?

Reply:
@mesutcank because of removing unused imports